packages/wallet/src/redux/protocols/channel-sync/readme.md

```diff
@@ -0,0 +1,26 @@
+# Channel Sync Protocol
```

Comment:
The purpose of this protocol is to handle the case where one player is a `commitment` or two behind another. This could happen if one player closes their browser while another player is sending a `commitment` to them.  The protocol simply has each player exchanging the `commitment`s they have and validating `commitments` from other players. Once the protocol is complete each player should be at the same `commitment` level.